app/controllers/projects_controller.rb

```diff
@@ -71,6 +71,7 @@ def volunteers
 
   def new
     @project = Project.new
+    @project.accepting_volunteers = true # default true
```

Comment:
Couldn't this be solved with a default in the migration?

Reply:
dang, you caught me being lazy